yarn-project/sequencer-client/src/block_builder/solo_block_builder.test.ts

```diff
@@ -237,9 +223,11 @@ describe('sequencer/solo_block_builder', () => {
     const newEncryptedLogs = new L2BlockL2Logs(txs.map(tx => tx.encryptedLogs || new TxL2Logs([])));
     const newUnencryptedLogs = new L2BlockL2Logs(txs.map(tx => tx.unencryptedLogs || new TxL2Logs([])));
 
+    // We are constructing the block here just to get body hash/calldata hash so we can pass in an empty archive and header
     const l2Block = L2Block.fromFields({
-      archive: rootRollupOutput.archive,
-      header: rootRollupOutput.header,
+      archive: AppendOnlyTreeSnapshot.zero(),
+      header: Header.empty(),
+      // Only the values bellow go to body hash/calldata hash
```

Comment:
```suggestion
      // Only the values below go to body hash/calldata hash
```

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4286/commits/29dfb8d024502df6a38d4dafb46de8b3890097ae